Shared/sdk/net/SyncStructures.h

```diff
@@ -565,6 +586,12 @@ struct SPlayerPuresyncFlags : public ISyncStructure
         bool bSyncingVelocity : 1;
         bool bStealthAiming : 1;
     } data;
+
+    // Add new ones in separate structs
+    struct
+    {
+        bool bIsReloadingWeapon : 1;
```

Comment:
Why not just put it into the existing `data` struct?

Reply:
I want to maintain compatibility with older BitStream version on the server/client.

Reply:
For this it's enough to permute BITCOUNT depending on a bitstream version.

Reply:
Earlier, I tried to apply it like this, but there were issues with comparing the bitstream versions with different client <-> server versions.